ams/AssetMigrator.cs

```diff
@@ -35,7 +35,16 @@ internal class AssetMigrator : BaseMigrator
         public override async Task MigrateAsync(CancellationToken cancellationToken)
         {
             var watch = Stopwatch.StartNew();
-            var account = await GetMediaAccountAsync(_options.AccountName, cancellationToken);
+            MediaServicesAccountResource? account = null;
+            try
+            {
+                account = await GetMediaAccountAsync(_options.AccountName, cancellationToken);
+            }
+            catch (Exception)
+            {
+                _logger.LogError("No valid media account was found.");
+                throw new Exception("No valid media account was found.");
+            }
```

Comment:
I don't think this change is needed for Storage account support.  This similar code can be moved to AssetAnalyzer to determine if the input account is for AMS or Storage account, but not put it here.